src/terms/xc.jl

```diff
@@ -8,7 +8,7 @@ struct Xc
     functionals::Vector{Libxc.Functional}
     scaling::Real  # to scale by an arbitrary factor (useful for exploration)
 end
-Xc(functionals::Vector{Libxc.Functional}; scaling=1) = Xc(functionals, scaling)  # default to no scaling
```

Comment:
Any reason why you removed this here. I would rather remove the type specialisation in `Vector{Symbol}` and keep it specific here as a guard to the type in the functionals field.

Reply:
Tests fail without it because it's called with Vector{Any}

Reply:
That smells like a bug somewhere else. Why was this not a problem before?

Reply:
Because Vector{Libxc.Functional}(arr) where arr is a Vector{Any} works if all terms in the vector are functionals, I would guess. As long as it works I'm happy, it's not as if it's going to be a subtle bug: either it works or it crashes